tests/utils.py

```diff
@@ -258,3 +260,67 @@ def check_signal_peaks(x, ref_peaks_pos, threshold=1):
     """
     peaks_pos = find_peaks(x, prominence=threshold)[0]
     np.testing.assert_equal(peaks_pos, ref_peaks_pos)
+
+
+def record_compartment_report(rep_conf: dict, target_manager: TargetManager):
```

Comment:
do you want to keep this only in the tests or do we want to offer this capability also to users? It may be useful. In that case we should stuff this in neurodamus itself no?

Reply:
It is only for testing. Users should only produce SONATA report.